### DISCUSSION TITLE
Morph integration is possible?

### DISCUSSION THREAD

**Discussion by avescarelo** [Roadhog360/Et-Futurum-Requiem#436] [Ideas]
### Initial Questions

- [X] I have searched this issue tracker and there is nothing similar already, and this is not on the [list of known issues](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/146). Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I have verified that this issue occurs in a SUPPORTED environment, meaning I can reproduce this WITHOUT OptiFine, and without Bukkit+Forge server software like KCauldron, Thermos, Crucible, Mohist etc

### Mod Version

2.5.1

### Describe the Issue

The Morph modification adds the ability to transform into different mobs. But due to the lack of integration with Et Futurum, players cannot use flight abilities, turning into flying mobs from the mod. Is it possible to implement this integration?

**Comment by Roadhog360** [comment]
This is a suggestion
